yazi-core/src/manager/commands/paste.rs

```diff
@@ -18,10 +18,11 @@ impl Manager {
 		let opt = opt.into() as Opt;
 
 		let dest = self.cwd();
-		if self.yanked.cut {
-			tasks.file_cut(&self.yanked, dest, opt.force);
-		} else {
+		if !self.yanked.cut {
 			tasks.file_copy(&self.yanked, dest, opt.force, opt.follow);
```

Comment:
Is the bug possibly cause of a missing `return` here?

Reply:
oops I missed it in reviewing, let me fix it

Reply:
fixed in https://github.com/sxyazi/yazi/commit/2f784e7ae71c07e02f318f9a6d3c45c586253e59

Reply:
Nice